tests/pipeline/StringContainerLibrary.cpp

```diff
@@ -53,7 +53,7 @@ class StringContainer : public Container<StringContainer> {
   }
 
   bool remove(const Target &Target) {
-    if (contains(Target))
+    if (not contains(Target))
```

Comment:
What was going on here?
Maybe this deserves its own commit.

Reply:
that file is just a test file, it is like fixing a failing test. i can push it in a commit of it's own but there is no much point to it.

Reply:
Bugfixes usually deserve their own commit, unless it's a "Rework heavily this part of the code" or "Fix many bugs" commit.
But I'll leave that to you.

Reply:
it was a wrong stub implementation of a method that never got tested and now it does

Reply:
Up to you, please comment whether you take action or not and mark as solved.